CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 
 # Change Log
 All notable changes to this project will be documented in this file.
- 
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2.2.post1] - 2021-05-05
```

Comment:
Why not just bump to `0.2.3`?

Reply:
because it doesn't touch any code, didn't feel the need to do a `0.2.3`

Reply:
🤷‍♂️ it still fixes a bug... I just think it's simpler to minimize the number of suffixes a project uses. Just `[major].[minor].[patch]` and maybe `alpha` and `beta`.